src/main/webapp/WEB-INF/views/login.jsp

```diff
@@ -0,0 +1,30 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%> 
+<%@taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
+<!-- <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"> -->
+<html>
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+		<title>Prajina Academy</title>
+	</head>
+	<body>
+		<form:form name="submitForm" method="POST">
```

Comment:
define an custom id for the forms.

Reply:
While JSP goes off the name property, standard HTML use of ids on form elements still good practice, and will come in handy for the later test suite.